pkg/controller/operators/catalog/step.go

```diff
@@ -140,7 +144,27 @@ func (b *builder) NewCRDV1Step(client apiextensionsv1client.ApiextensionsV1Inter
 					currentCRD, _ := client.CustomResourceDefinitions().Get(context.TODO(), crd.GetName(), metav1.GetOptions{})
 					crd.SetResourceVersion(currentCRD.GetResourceVersion())
 					if err = validateV1CRDCompatibility(b.dynamicClient, currentCRD, crd); err != nil {
-						return fmt.Errorf("error validating existing CRs against new CRD's schema for %q: %w", step.Resource.Name, err)
+						vErr := &validationError{}
+						// if the conversion strategy in the new CRD is not "Webhook" OR the error is not a ValidationError
+						// return an error. This will catch and return any errors that occur unrelated to actual validation.
+						// For example, the API server returning an error when performing a list operation
+						if crd.Spec.Conversion == nil || crd.Spec.Conversion.Strategy != apiextensionsv1.WebhookConverter || !errors.As(err, vErr) {
+							return fmt.Errorf("error validating existing CRs against new CRD's schema for %q: %w", step.Resource.Name, err)
+						}
+						// If the conversion strategy in the new CRD is "Webhook" and the error that occurred
+						// is an error related to validation, warn that validation failed but that we are trusting
+						// that the conversion strategy specified by the author will successfully convert to a format
+						// that passes validation and allow the upgrade to continue
+						warnTempl := `validation of existing CRs against new CRD's schema failed, but a webhook conversion strategy was specified.
+allowing the CRD upgrade to occur as we can't run the webhook, but is assumed that it will successfully convert existing CRs
+to a format that would have passed validation.
+
+CRD: %q
+Validation Error: %s
+`
```

Comment:
```suggestion
						warnTempl := `Validation of existing CRs against the new CRD's schema failed, but a webhook conversion strategy was specified in the new CRD. The new webhook will only start after the bundle is upgraded, so we must assume that it will successfully convert existing CRs to a format that would have passed validation.

CRD: %q
Validation Error: %s
`
```

Reply:
Updated in https://github.com/operator-framework/operator-lifecycle-manager/pull/3209/commits/91b7f5fe469059b97bd3d0803605b912b96af344